app/scaffold-controllers/navigationController.js

```diff
@@ -11,17 +13,26 @@ function(
     Application,
     AnchoredLayoutPlugin,
     NavigationPlugin,
-    NavigationHeaderController
+    NavigationHeaderController,
+    SearchBarController,
+    SearchConfig
```

Comment:
I think cleaner is better - just that there is already precedent from the "HeaderConfig" to do it this way. Alternatively we could have a default config that the controller brings in, but maybe we could override that behaviour externally (to cover the different use case scenario) - thoughts on that?